src/components/daydream-controls.js

```diff
@@ -0,0 +1,204 @@
+var registerComponent = require('../core/component').registerComponent;
+var bind = require('../utils/bind');
+var isControllerPresent = require('../utils/tracked-controls').isControllerPresent;
+
+var DAYDREAM_CONTROLLER_MODEL_BASE_URL = 'https://cdn.aframe.io/controllers/google/';
+var DAYDREAM_CONTROLLER_MODEL_OBJ_URL = DAYDREAM_CONTROLLER_MODEL_BASE_URL + 'vr_controller_daydream.obj';
+var DAYDREAM_CONTROLLER_MODEL_OBJ_MTL = DAYDREAM_CONTROLLER_MODEL_BASE_URL + 'vr_controller_daydream.mtl';
+
+var GAMEPAD_ID_PREFIX = 'Daydream Controller';
+
+/**
+ * Vive Controls Component
+ * Interfaces with vive controllers and maps Gamepad events to
+ * common controller buttons: trackpad, trigger, grip, menu and system
+ * It loads a controller model and highlights the pressed buttons
+ */
+module.exports.Component = registerComponent('daydream-controls', {
+  schema: {
+    hand: {default: 'right'}, // This informs the degenerate arm model.
```

Comment:
Just a single entity + controller component:

```html
<a-entity daydream-controls></a-entity>
```

Should default to whatever `getGamepad.hand` returns, unless the `hand` property is specified, in which case it overrides.   Sound about right? @dmarcos?

As far as `hand-controls`.   The arm model puts the hand too close into the body and leaves the other hand floating.   The Daydream controller works sufficiently different enough from VIVE and Touch, I question trying to make it work with hand-controls at all.   At the very least, we would want to tweak the arm model so that it is more useable as a hand.

The laser pointer type control would probably be the better primary control model for daydream-controller (rather than hand-controls):

```html
<a-entity daydream-controller visible='false'>
   <a-cone id="ray" color="cyan" position="0 0 -2" rotation="-90 0 0" radius-bottom="0.005" radius-top="0.001" height="4"></a-cone>
</a-entity>
```